src/utils/mtev_lfu.c

```diff
@@ -78,19 +83,25 @@ hs_init(ck_hs_t *hs, unsigned int mode, ck_hs_hash_cb_t *hf, ck_hs_compare_cb_t
 }
 
 static bool
-hs_string_compare(const void *a, const void *b)
+hs_lfu_key_compare(const void *a, const void *b)
 {
-  return strcmp((const char * const)a, (const char * const)b) == 0;
+  struct lfu_key *left = (struct lfu_key *)a;
+  struct lfu_key *right = (struct lfu_key *)b;
```

Comment:
nit (and I know it's a little silly), but would prefer if the vars and the typecasts all said `const`.